docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3081,11 +3093,13 @@ Following rule will delete all `script` elements with a `banner` substring in th
 $$script[tag-content="banner"]
 ```
 
-**Nested elements**
+#### `wildcard`
 
-If we are dealing with multiple nested elements and they all fall within the same HTML filtering rule, they all are going to be deleted.
+:::caution Deprecation notice
 
-* **`wildcard`**
+The use of this special attribute is deprecated in favor of the `:contains()` pseudo-class.
```

Comment:
but it is not deprecated for the extension yet

Reply:
@ameshkov What if we remove the deprecation notices until all products implement the new syntax?

Reply:
The version numbers where support will be discontinued can be specified.

Reply:
I've reworded these notices.